src/utils.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Collection of helper methods for Superset Charm."""
+
+import logging
+import os
+import secrets
+import string
+
+logger = logging.getLogger(__name__)
+
+
+def charm_path(file_path):
+    """Get path for Charm.
+
+    Args:
+        file_path: charm file_path
+
+    Returns:
+        path: full charm path
+    """
+    charm_dir = os.path.abspath(
+        os.path.join(os.path.dirname(__file__), os.pardir)
+    )
+    path = os.path.join(charm_dir, file_path)
+    return path
+
+
+def generate_password() -> str:
+    """Create randomized string for use as app passwords.
+
+    Returns:
+        String of 32 randomized letter+digit characters
+    """
+    return "".join(
+        [
+            secrets.choice(string.ascii_letters + string.digits)
+            for _ in range(32)
+        ]
+    )
+
+
+def read_file(file_path):
```

Comment:
Do you need this if you are not doing any additional work?

Reply:
Agree with you, I've moved these lines to the `push_files` function instead of a dedicated function.